data/sqlutil/dynamic_frame.go

```diff
@@ -10,13 +10,25 @@ import (
 
 const STRING = "string"
 
-func isDynamic(converters []Converter) bool {
-	for _, conv := range converters {
+func isDynamic(converters []Converter) (bool, []Converter) {
```

Comment:
nit: I would call this `filterDynamic` since it's doing now more than just checking a type.